app/src/main/kotlin/jp/co/yumemi/android/code_check/RepositorySearchFragment.kt

```
@@ -0,0 +1,93 @@
+        }
+    }
+
+    /**
+     * 検索処理の実行
+     */
+    private fun performSearch(query: String) {
+        lifecycleScope.launch {
+            try {
+                val results = viewModel.fetchSearchResults(query)
+                adapter.submitList(results)
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)